src/components/Pages/AboutMe/Aboutme.jsx

```diff
@@ -1,18 +1,19 @@
 import { Button } from "react-bootstrap";
-import "../Maincontainer.css";
+import styled from "styled-components";
+import "../../Maincontainer.css";
 import SectionHeading from "../SectionHeading";
 const Aboutme = () => {
   return (
     <>
-      <div className="row p-5">
+      <Div className="row p-5">
```

Comment:
Use a meaningful component. Instead of `Div`, name like `SectionContainer` would have more meaning.

Reply:
Done